hedy.py

```diff
@@ -1748,6 +1760,7 @@ def preprocess_blocks(code, level):
                 raise hedy.exceptions.LockedLanguageFeatureException(concept="nested blocks")
 
         if next_line_needs_indentation and current_number_of_indents <= previous_number_of_indents:
+            fixed_code = program_repair.fix_indent(code, line_number - 1, leading_spaces, indent_size)
```

Comment:
If we always subtract - from the line number, maybe we can do that in the method?